docs/contributing.md

```diff
@@ -0,0 +1,36 @@
+# How to contribute to Azure TRE
+
+First of all thank you. Azure TRE is a collective effort to make it easier to create and manage trusted research environment.
```

Comment:
Not sure need this sentence? Maybe would be useful to have a link to the developer docs, as we now have them.

Reply:
If look at contributing guides for other OSS projects they typically describe how to clone the repo etc.

Maybe need to link to https://microsoft.github.io/AzureTRE/using-tre/local-development/local-development/

Although is a bit confusing that we have a contributing section visible here too: https://microsoft.github.io/AzureTRE/tre-developers/api/